test.py

```diff
@@ -134,40 +65,13 @@ def run_cmd(cmd):
     return retcode
 
 
-def run_go_fmt(opts):
-    return run_cmd(['go', 'fmt', './...'])
-
-
-def run_go_build(opts):
-    return run_cmd(['go', 'build', './...'])
-
-
```

Comment:
can we add these targets to the Makefile?

A default make target that runs static-analysis and go build would be nice.  The default make target as static-analysis is unusual.

Reply:
I don't think we need the `build`, as there is nothing to build (I think the python code was copied from other projects). `go fmt` is an IDE thing, that is picked and checked via the static analysis now, so is redundant as well.